src/components/elements/Map-mapbox/Map.tsx

```diff
@@ -257,22 +262,56 @@ export const MapContainer = ({
     }
   };
 
+  const flyToPolygonBounds = async (poly_id: string) => {
+    const bbox = await fetchGetV2TerrafundPolygonBboxUuid({ pathParams: { uuid: poly_id } });
+    const bounds: any = bbox.bbox;
+    if (!map.current) {
+      return;
+    }
+    map.current.fitBounds(bounds, {
+      padding: 100,
+      linear: false
+    });
+  };
+
   const onSaveEdit = async () => {
     if (map.current && draw.current) {
       const geojson = draw.current.getAll();
       if (geojson) {
         if (polygonFromMap?.uuid) {
+          !pdView && onCancelEdit();
           const feature = geojson.features[0];
           const response = await fetchPutV2TerrafundPolygonUuid({
-            body: { geometry: JSON.stringify(feature) },
+            body: {
+              geometry: JSON.stringify(feature),
+              // @ts-ignore
+              adminUpdate: pdView ? false : true
+            },
             pathParams: { uuid: polygonFromMap?.uuid }
           });
           reloadSiteData?.();
-          if (response.message == "Geometry updated successfully.") {
+          if (
+            response.message == "Site polygon version created successfully." ||
+            response.message == "Geometry updated successfully."
```

Comment:
Please modify backend response, use status code 201 when an entity is created and 200 when an entity is updated.
Later come here and use those status codes for the conditional.
Right now it is depending in synchronizing the messages between BE and FE

Reply:
Why do we have an endpoint that can both create and update? These are separate REST actions and should be separate endpoints.